Description: Upstream changes introduced in version 0.3.6-0ubuntu2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 indicator-session (0.3.6-0ubuntu2) oneiric; urgency=low
 .
   * data/indicator-session.convert
     - Fixed improperly mapped gconf to gsettings keys (LP: #858798)
 .
 The person named in the Author field signed this changelog entry.
Author: Ken VanDine <ken.vandine@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/858798

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- indicator-session-0.3.6.orig/data/indicator-session.convert
+++ indicator-session-0.3.6/data/indicator-session.convert
@@ -1,5 +1,5 @@
 [com.canonical.indicator.session]
-suppress_logout_restart_shutdown = /apps/indicator-session/suppress-logout-restart-shutdown
-suppress_logout_menuitem = /apps/indicator-session/suppress-logout-menuitem
-suppress_restart_menuitem = /apps/indicator-session/suppress-restart-menuitem
-suppress_shutdown_menuitem = /apps/indicator-session/suppress-shutdown-menuitem
+suppress-logout-restart-shutdown = /apps/indicator-session/suppress_logout_restart_shutdown
+suppress-logout-menuitem = /apps/indicator-session/suppress_logout_menuitem
+suppress-restart-menuitem = /apps/indicator-session/suppress_restart_menuitem
+suppress-shutdown-menuitem = /apps/indicator-session/suppress_shutdown_menuitem
